### PR TITLE
update(Prison): Negative Time Fix

### PR DESCRIPTION
**Pull request by Savage912-Official**
Prison jobs will only reduce your time if you have 5 or more months remaining on your sentence.

### CODE REVIEW

File: resources/usa_prison/jobs/janitor/sv_janitor.lua [hunk 1]
```diff
@@ -60,10 +60,12 @@ AddEventHandler("prison-janitor:toggleJob", function(targetSrc)
                 payment = payment + PAY_RATES[currentlyCleaningTaskName[src]]
             end
         end
-        char.set("jailTime", time)
+        if time > 5 then
+            char.set("jailTime", time)
+            TriggerClientEvent("usa:notify", src, "Reduced Prison Time, New Time: " .. time)
+        end
```
Comment (minipunch) [inline resources/usa_prison/jobs/janitor/sv_janitor.lua:66]:
```suggestion
       char.set("jailTime", math.max(0, time))
```

Comment (minipunch) [reply resources/usa_prison/jobs/janitor/sv_janitor.lua:66]: That basically means it will always be >= 0

**Review by minipunch — CHANGES_REQUESTED** [review]

File: resources/usa_prison/jobs/janitor/sv_janitor.lua (hunk 1 above)
Comment (Savage912-Official) [reply resources/usa_prison/jobs/janitor/sv_janitor.lua]: Easy enough, didn't think of it that way. Learn new things everyday.

File: resources/usa_prison/jobs/janitor/sv_janitor.lua [hunk 2]
```diff
@@ -60,10 +60,9 @@ AddEventHandler("prison-janitor:toggleJob", function(targetSrc)
                 payment = payment + PAY_RATES[currentlyCleaningTaskName[src]]
             end
         end
-        char.set("jailTime", time)
+       char.set("jailTime", math.max(0, time))
         char.giveBank(payment)
         TriggerClientEvent("usa:notify", src, "You've been paid: $" .. exports.globals:comma_value(payment))
-        TriggerClientEvent("usa:notify", src, "Reduced Prison Time, New Time: " .. time)
```
Comment (minipunch) [inline resources/usa_prison/jobs/janitor/sv_janitor.lua:66]: Did you mean to remove this?

Comment (Savage912-Official) [reply resources/usa_prison/jobs/janitor/sv_janitor.lua:66]: Nah it was removed when I did the other commit before I commited yours and didn't notice it.

**Review by minipunch — APPROVED** [review]

**Review by minipunch — APPROVED** [review]

**Pull request merged**